packages/react/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": ["../internal-tooling/eslint/index.js"],
+  "overrides": [
+    {
+      "files": "**/icons/*.tsx",
+      "rules": {
+        "react/prop-types": "off"
```

Comment:
ESLint with React plugin handles these icons as React components, it will be very strange to have there `propTypes`...